launcher-implementation/src/main/input_resources/sbt/sbt.boot.properties

```diff
@@ -12,8 +12,8 @@
 
 [repositories]
   local
+  jcenter: https://jcenter.bintray.com/
 ${{repositories}}
-  maven-central
```

Comment:
Can we remove maven-central here? I think there are some situations in which jcenter is not complete.